calc/vxlan_resolver.go

```diff
@@ -693,10 +693,12 @@ func (c *VXLANResolver) vtepMACForHost(nodename string) string {
 	}
 
 	hasher := sha1.New()
-	hasher.Write([]byte(nodename))
+	_, err := hasher.Write([]byte(nodename))
+	if err != nil {
+		logrus.WithField("node", nodename).Debug("Failed to write hash for node")
```

Comment:
Remember to do `logrus.WithError(err)` to actually log out the error.  Also, you should log this at Error (or maybe even Panic) since hasher.Write shouldn't ever return an error (IIRC it's only defined that way to allow it to implement the Writer interface).

Reply:
Fixed